### PR TITLE
#1109 Fix SVarC init for voltage reference and mode

### PR DESCRIPTION
**Pull request by MarianneSaugierRTE**
*The selection of the reference URefPu depended on Mode0: it should depend on modeAuto or modeManual.
*The way we start between modeAuto and modeManual should not be hardcoded.
*In the nrt the SVarC should  be in modeManual for the votlage setpoint changes.
=> With these corrections the simulation is well initialized if we start from a dump (or not).

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard.mo [hunk 1]
```diff
@@ -72,7 +72,7 @@ protected
   final parameter Types.PerUnit BVar0Pu = B0Pu - BShuntPu "Start value of variable susceptance in p.u";
   parameter BaseControls.Mode Mode0 "Start value for mode";
   parameter Boolean selectModeAuto0 = true "Start value of the boolean indicating whether the SVarC is initially in automatic configuration";
-  parameter Integer setModeManual0 = 2 "Start value of the mode when in manual configuration";
+  final parameter Integer setModeManual0 = Integer(Mode0) "Start value of the mode when in manual configuration";
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard.mo:75]: Should'nt it be changed also for SVarCPV.mo ?

File: nrt/data/SVarC/SVarC_1_StepUref/SVarC.par [hunk 2]
```diff
@@ -50,6 +50,7 @@
     <par type="DOUBLE" name="UThresholdDown" value="218"/>
     <par type="DOUBLE" name="tThresholdUp" value="60"/>
     <par type="DOUBLE" name="tThresholdDown" value="0"/>
+    <par type="BOOL" name="selectModeAuto0" value="false"/>
```
Comment (cossartque) [inline nrt/data/SVarC/SVarC_1_StepUref/SVarC.par:53]: Same, shouldn't it be changed also for SVarCPV ?

File: nrt/data/SVarCPV/SVarC_1_StepUref/reference/outputs/curves/curves.csv [hunk 3]
```diff
@@ -1,6 +1,6 @@
 time;Test_svarc_UPu;Test_svarc_BPu;Test_svarc_QInjPu;Test_svarc_modeHandling_mode;Test_svarc_modeHandling_URefPu;
 0.000000;1.000000;0.000000;0.000000;3.000000;1.000000;
 10.000000;1.000000;0.000000;0.000000;3.000000;1.000000;
```
Comment (cossartque) [inline nrt/data/SVarCPV/SVarC_1_StepUref/reference/outputs/curves/curves.csv:3]: Same, shouldn't it be changed also for SVarCPV ?

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard.mo (hunk 1 above)
Comment (MarianneSaugierRTE) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/StaticVarCompensators/SVarCStandard.mo:75]: You're right!! Done. Sorry I forgot to commit the corrections I made at last when I noticed there was a problem with SVarCPV. Thanks!

**Review by cossartque — APPROVED** [review]
It's ok for me :)

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**